angular/core/components/group_page/group_theme/group_theme.coffee

```diff
@@ -34,3 +36,12 @@ angular.module('loomioApp').directive 'groupTheme', ->
 
     $scope.logoHoverOut = ->
       $scope.logoHover = false
+
+    $scope.triggerThemeHover = ->
+      $('.group-theme__trigger-upload').hover ->
```

Comment:
we won't want to use jQuery anymore (I thought we took it out actually), instead we want

```
document.querySelector('selector')
```